### PR TITLE
Enrich package.json

### PR DESCRIPTION
**Pull request by PeterDaveHello**
Add homepage, keywords and repository info in package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,5 +27,9 @@
     "babelify": "^7.3.0",
     "browserify": "^13.1.0",
     "watchify": "^3.7.0"
+  },
```
Comment (amilajack) [inline package.json:31]: Seems like the indentation is off here

Comment (PeterDaveHello) [reply package.json:31]: Fixed, thanks.

File: package.json [hunk 2]
```diff
@@ -12,7 +12,13 @@
     "dev": "watchify src/index.js -t babelify --standalone Grade -o docs/dist/grade.js",
     "prepublish": "npm run build"
   },
-  "keywords": [],
+  "keywords": [
+    "complimentary",
```
Comment (benhowdle89) [inline package.json:16]: This needs to be "complementary" 👍

Comment (PeterDaveHello) [reply package.json:16]: Updated.

**Pull request merged**